app/controllers/identify_controller.rb

```diff
@@ -65,6 +65,16 @@ def client_cert
 
   def process_cert(raw_cert)
     cert = Certificate.new(OpenSSL::X509::Certificate.new(raw_cert))
+    logger.info({
+      name: 'Certificate Processed',
+      signing_key_id: cert.signing_key_id,
+      key_id: cert.key_id,
+      issuer: cert.issuer.to_s,
+      card_type: cert.card_type,
+      valid_policies: cert.valid_policies?,
+      valid: cert.valid?(is_leaf: true),
+      error: !cert.valid?(is_leaf: true) ? cert.validate_cert(is_leaf: true) : nil,
```

Comment:
since we do `cert.valid?` twice, WDYT of extracting into a variable? I'm not sure how expensive of an op it is
```suggestion
      valid: cert_valid,
      error: !cert_valid ? cert.validate_cert(is_leaf: true) : nil,
```

Reply:
I think it's a good idea. The validation is a bit awkward so I pulled the result into a variable above this.